addons/website/tests/test_converter.py

```diff
@@ -105,3 +106,9 @@ def test_all(self):
             "do-you-know-martine-a-la-plage",
             self._slugify("Do YOU know 'Martine à la plage' ?")
         )
+
+    def test_non_ascii_chars(self):
+        self.assertEqual(
+            unicodedata.normalize('NFKD', "测试시험"),
+            self._slugify("测试시험")
+        )
```

Comment:
Make your test as simple as possible, here you're calling normalize(), I don't wanna go and read the unicode spec to understand how NFKD normalisation is gonna change 测试시험.

Also it could be interesting to have many more tests, one for each case, like one for the NFKD normalisation, one for merging successive -, one for removing punctuation, one for striping the -, ... You can use `subTest` for that.

Reply:
As it is testing base, ideally the unittest should be in base too. But I don't know if we have tests for this function in base... If not testing it in website is fine... I guess.